GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -744,8 +744,9 @@ extension ChatViewModel {
 
 extension ChatViewModel {
     private func sendChoiceCardResponse(_ option: ChatChoiceCardOption, to messageId: String) {
-        guard let value = option.value else { return }
-        environment.sendSelectedOptionValue(value) { [weak self] result in
+
+        guard let option = option.asSingleChoiceOption() else { return }
+        environment.sendSelectedOptionValue(option) { [weak self] result in
```

Comment:
Let's add test ensuring that new `environment.sendSelectedOptionValue` is called. I suppose the easiest way to test it is to make `sendChoiceCardResponse` internal instead of `private`.

Reply:
I added a lot of mocks, but now this case covered by test :)
fixed - 5f91fc53ba4437584fe7446ecf1f42ecc2dd4ca2